### DISCUSSION TITLE
any chance to dim Sonoff t3 led boxes

### DISCUSSION THREAD

**Discussion by NEoKhajitt** [arendst/Tasmota#10268] [Q&A] [answered]
This might be a real shot in the dark but, is there any possibility to dim the led boxes on the sonoff t3 devices ? I recently installed one in my bedroom and it's facing directly at my bed and the light does wake me up at night.
Just to clarify, not the led indictor light the flashes on connections ect. the actually 3 boxes to indicate where the buttons are.

**Comment by sfromis** [comment, marked as answer]
Unless the guide lights have gpio pins (which they do not in a 2 gang variant [template](https://templates.blakadder.com/sonoff_TX-T3EU2C) I found), Tasmota cannot control what happens there. In that case, you'd have to rewire the wall switch, do something inside to reduce light, or cover with some dark tape. I've used nail varnish to dim a few leds; while that may not be a good fit for the smooth surface, it could make sense inside the disturbing thing.

**Reply by NEoKhajitt** [reply]
i see the dilemma. 
This is the [switch](https://templates.blakadder.com/sonoff_TX-T3US3C.html) that i have.
ok, for this specific switch i might consider coating the inside with some nail varnish, the reason why i was asking if possible to dim, it would have been nice to create a automation where the leds dim after a specific time eg. 11PM and return to normal brightness on sun.sun.above_horizon
